src/cli/init.rs

```diff
@@ -149,9 +153,42 @@ const GITIGNORE_TEMPLATE: &str = r#"
 *.egg-info
 "#;
 
-const GITATTRIBUTES_TEMPLATE: &str = r#"# GitHub syntax highlighting
+enum GitAttributes {
+    GitHub,
+    GitLab,
+    Codeberg,
+}
+
+impl GitAttributes {
+    fn template(&self) -> &'static str {
+        match self {
+            GitAttributes::GitHub => {
+                r#"# GitHub syntax highlighting
```

Comment:
Lets simplify this by moving multiple options under the same result. 

```suggestion
                r#"# SCM syntax highlighting
```

Reply:
I'll put Codeberg and Github under the same result.